pkg/annotations/annotations.go

```diff
@@ -74,7 +74,7 @@ const (
 	PlatformRuntimePath = "io.kubernetes.cri-o.PlatformRuntimePath"
 
 	// SeccompProfileAnnotation can be used to set the seccomp profile for a specific container, pod or whole image.
-	SeccompProfileAnnotation = "io.kubernetes.cri-o.seccompProfile"
+	SeccompProfileAnnotation = "seccomp-profile.kubernetes.cri-o.io"
```

Comment:
Kubernetes expects public annotations to have a `/` in them.

Reply:
Everytime? Are multiple allowed, like `kubernetes.cri-o.io/seccomp-profile/<CONTAINER>` for container specific annotations) and `kubernetes.cri-o.io/seccomp-profile` for pods/images?

Reply:
Just one `/` please.

Reply:
If they don't have a `/` in them, they are considered cluster private (ie, they are reserved for a cluster adminstrator to assign; tools and public code should try not to do that).

Reply:
So we could say that we allow:
- `seccomp-profile.kubernetes.cri-o.io/POD` for the whole pod
- `seccomp-profile.kubernetes.cri-o.io/<CONTAINER>` for a specific container name

The same would apply to the image annotation then, which should probably default to `POD`?

Reply:
That may work. Do we know that `POD` isn't a valid name for a container?

Reply:
@sftim it is not:
```
The Pod "test-pod" is invalid: spec.containers[0].name: Invalid value: "POD": a lowercase RFC 1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')
```